sci-electronics/geda/geda-1.9.2-r1.ebuild

```diff
@@ -1,46 +1,47 @@
 # Copyright 1999-2020 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=6
-inherit autotools eutils xdg-utils gnome2-utils versionator
+EAPI=7
+
+inherit autotools xdg
 
 MY_PN=${PN}-gaf
 MY_P=${MY_PN}-${PV}
 
 DESCRIPTION="GPL Electronic Design Automation (gEDA):gaf core package"
 HOMEPAGE="http://wiki.geda-project.org/geda:gaf"
-SRC_URI="http://ftp.geda-project.org/${MY_PN}/unstable/v$(get_version_component_range 1-2)/${PV}/${MY_P}.tar.gz"
+SRC_URI="http://ftp.geda-project.org/${MY_PN}/unstable/v$(ver_cut 1-2)/${PV}/${MY_P}.tar.gz"
 
 LICENSE="GPL-2"
 SLOT="0"
 KEYWORDS="amd64 ppc x86"
 IUSE="debug doc examples nls stroke threads"
 
-CDEPEND="
+RDEPEND="
 	dev-libs/glib:2
+	dev-scheme/guile
+	sci-electronics/electronics-menu
+	x11-libs/cairo
+	x11-libs/gdk-pixbuf
 	x11-libs/gtk+:2
 	x11-libs/pango
-	>=x11-libs/cairo-1.2.0
-	x11-libs/gdk-pixbuf
-	>=dev-scheme/guile-2.0.0
 	nls? ( virtual/libintl )
-	stroke? ( >=dev-libs/libstroke-0.5.1 )"
+	stroke? ( dev-libs/libstroke )"
 
-DEPEND="${CDEPEND}
-	sys-apps/groff
+DEPEND="${RDEPEND}
 	dev-util/desktop-file-utils
-	x11-misc/shared-mime-info
+	sys-apps/groff
+	x11-misc/shared-mime-info"
```

Comment:
check these, groff a `DEPEND` seems fishy...

Reply:
Ok, switched it to BDEPEND